react-native-in-app-payments-quickstart/app/Utilities.js

```diff
@@ -26,10 +26,14 @@ export function uuidv4() {
   });
 }
 
-export function printCurlCommand(nonce) {
+export function printCurlCommand(nonce, appId) {
+  let hostUrl = 'https://connect.squareup.com';
+  if (appId.startsWith('sandbox')) {
+    hostUrl = 'https://connect.squareupsandbox.com';
```

Comment:
Suggest to add a comment to describe sandbox vs production endpoints url.

Can you do: `const hostUrl = appId.startsWith('sandbox') ? ... : ...`?

Reply:
Updated